File: backend/java/localpos-user-service/src/main/java/com/pds/localpos/userservice/UserServiceApplication.java
package com.pds.localpos.userservice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {
        "com.pds.localpos",
        "com.pds.localpos.security"
})
public class UserServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(UserServiceApplication.class, args);
    }

}

File: backend/kotlin/localpos-user-service-kt/src/main/kotlin/com/pds/localpos/userservice/UserServiceApplication.kt
package com.pds.localpos.userservice

import com.pds.localpos.userservice.config.CorsProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.pds.localpos"])
@EnableConfigurationProperties(CorsProperties::class)
class UserServiceApplication

fun main(args: Array<String>) {
	runApplication<UserServiceApplication>(*args)
}
